app/forms/mappings/LocalDateFormatter.scala

```diff
@@ -22,14 +22,14 @@ import play.api.data.format.Formatter
 import java.time.LocalDate
 import scala.util.{Failure, Success, Try}
 
-private[mappings] class LocalDateFormatter(
-                                            invalidKey: String,
-                                            allRequiredKey: String,
-                                            twoRequiredKey: String,
-                                            requiredKey: String,
-                                            yearMinDigitKey: String,
-                                            args: Seq[String] = Seq.empty
-                                          ) extends Formatter[LocalDate] with Formatters {
+protected[mappings] class LocalDateFormatter(
```

Comment:
We can simplify it
Ref: https://github.com/hmrc/hmrc-frontend-scaffold.g8/blob/main/src/main/g8/app/forms/mappings/LocalDateFormatter.scala